### DISCUSSION TITLE
Is it possible to get auto update from main repo ?

### DISCUSSION THREAD

**Discussion by ericjack371** [Schweinepriester/github-profile-achievements#949] [Q&A] [answered]
i forked a repo. the main repo update everyday.. 
so is it possible to get auto update from the main repo to my repo ?

**Comment by AFN4NX** [comment, marked as answer]
yes you can.

**Reply by ericjack371** [reply]
how to do that

**Reply by AFN4NX** [reply]
you have to add a auto upstream.yml on workflow

**Comment by Athexdev** [comment]
add a .yml folw in your workflow
